aslam_cv_pipeline/src/visual-npipeline.cc

```diff
@@ -70,24 +71,39 @@ bool VisualNPipeline::processImageBlockingIfFull(size_t camera_index, const cv::
   return false;
 }
 
+bool VisualNPipeline::processImageNonBlockingDroppingOldestNFrameIfFull(
+    size_t camera_index, const cv::Mat &image, int64_t timestamp,
+    size_t max_output_queue_size) {
+  CHECK_GE(max_output_queue_size, 1u);
+
+  bool result = false;
```

Comment:
result -> image_dropped?

Reply:
this will change: https://github.com/ethz-asl/aslam_cv2/pull/7/files